packages/xo-web/src/xo-app/home/index.js

```diff
@@ -650,11 +662,26 @@ export default class Home extends Component {
 
     const sort = this._getDefaultSort(props)
 
+    const {
+      $container: selectedHosts,
+      $pool: selectedPools,
+      power_state: powerStates,
+      resourceSet: selectedResourceSets,
+      tags: selectedTags,
+    } = properties
```

Comment:
What's the point of destructuring if you inject each property right below?

Reply:
I use `powerStates` twice  ^_^